Tests/ExamplesTest/expected_output/httpanalyzer_manyprotocols.txt

```diff
@@ -137,4 +137,4 @@ Content-type count
 | text/javascript                | 6     |
 | text/plain                     | 1     |
 | text/xml                       | 2     |
-------------------------------------------
```

Comment:
This change is not necessary, right?

Reply:
I did some unintended changes in the commit history due to my IDE settings and commit organization issues (e.g. I tried to rebase onto dev in my fork, that is why you saw the messy history). I'll review the changes in this patch and can create a fresh pull request if preferred.

Reply:
I don't think a new pull request is necessary. You can just revert the change in this file